widgetry/src/widgets/button.rs

```diff
@@ -611,10 +612,7 @@ impl<'b, 'a: 'b, 'c> ButtonBuilder<'a, 'c> {
 
                 let text = label.text.clone().or(default.and_then(|d| d.text.clone()));
 
-                // Is there a better way to do this like a `guard let`?
```

Comment:
I don't think this one is very clear. I guess we could just delete this, and the subsequent line and change 612 to:

```diff
- let text = label.text.clone().or(default.and_then(|d| d.text.clone()));
+ let text = label.text.clone().or(default.and_then(|d| d.text.clone()))?;
```

Reply:
Good point!